uktrade/data-hub-frontend
# feature/CLS2-326-display-filter-message-for-large-trees

## Description of change

When a company contains a large number of companies in their dnb family tree, it is not possible to search on subsidiaries as there is too much data.

Using the related companies count endpoint, we will disable the option to include subsidiaries and show a message explaining why

## Test instructions

Using a company with a large family tree, for example EDF, and go to the investments page http://localhost:3000/companies/f8ed2f41-b7d7-45f7-977a-2b3615ffa753/investments/projects?page=1&sortby=created_on%3Adesc. You will see the subsidiaries checkbox disabled

## Screenshots

### Before

![image](https://github.com/uktrade/data-hub-frontend/assets/102232401/ddd8f113-3e36-46a5-9e9f-3f0c40be786f)

### After

![image](https://github.com/uktrade/data-hub-frontend/assets/102232401/8e07fca5-0cd4-4aed-9036-9f5e2f95d7dc)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
